src/bci_build/package/python.py

```diff
@@ -45,14 +45,20 @@ def _get_python_kwargs(py3_ver: _PYTHON_VERSIONS, os_version: OsVersion):
     py3_ver_replacement = f"%%py{py3_ver_nodots}_ver%%"
     pip3 = f"{py3}-pip"
     pip3_replacement = "%%pip_ver%%"
-    has_pipx = False
-    has_wheel = True if is_system_py or os_version == OsVersion.TUMBLEWEED else False
-    # Tumbleweed rocks
-    if os_version == OsVersion.TUMBLEWEED:
-        has_pipx = True
-    # Enabled only for Python 3.11 on SLE15 (jsc#PED-5573)
-    if os_version not in (OsVersion.BASALT, OsVersion.TUMBLEWEED) and py3_ver == "3.11":
+    has_pipx: bool = False
+    has_wheel: bool = is_system_py
```

Comment:
Do we have to set these? We cover all cases in the `if - elif` below

Reply:
not right now, but for basalt/slcc/slac we would. I can remove them for now.

Reply:
actually I can not remove them because these are local variables not the dataclass attributes.